TestIndexFixtures test is failing

First run yielded:

```
2022/12/26 14:25:45 [margaret/indexes/badger] overwrote batch limit 0
=== RUN   TestIndexFixtures
    indexspeed_test.go:108:
                Error Trace:    /home/go-ssb/go-ssb/multilogs/indexspeed_test.go:108
                Error:          Not equal:
                                expected: 100000
                                actual  : 0
                Test:           TestIndexFixtures
                Messages:       case TestIndexFixtures has wrong number of messages
--- FAIL: TestIndexFixtures (1.85s)
FAIL
FAIL    github.com/ssbc/go-ssb/multilogs        1.873s
FAIL
```

After that, results seem to be consistently like this:

```
2022/12/26 14:27:17 [margaret/indexes/badger] overwrote batch limit 0
=== RUN   TestIndexFixtures
    indexspeed_test.go:92: + dest=testrun/TestIndexFixtures/log
        + test testrun/TestIndexFixtures/log '!=' ''
        + sha256sum -c v2-sloop-m100000-a2000.tar.gz.shasum
        v2-sloop-m100000-a2000.tar.gz: OK
        + rm -r tmp
        rm: cannot remove 'tmp': No such file or directory
        + rm -r testrun
        + mkdir -p tmp/unpack
        + tar xf v2-sloop-m100000-a2000.tar.gz -C tmp/unpack
        + go run go.cryptoscope.co/ssb/cmd/ssb-offset-converter -if lfo tmp/unpack/flume/log.offset testrun/TestIndexFixtures/log
        no required module provides package go.cryptoscope.co/ssb/cmd/ssb-offset-converter; to add it:
                go get go.cryptoscope.co/ssb/cmd/ssb-offset-converter
        
    indexspeed_test.go:93: 
                Error Trace:    /home/go-ssb/go-ssb/multilogs/indexspeed_test.go:93
                Error:          Received unexpected error:
                                exit status 1
                Test:           TestIndexFixtures
--- FAIL: TestIndexFixtures (1.58s)
FAIL
FAIL    github.com/ssbc/go-ssb/multilogs        1.606s
FAIL
```

--- a/multilogs/indexspeed_test.go
+++ b/multilogs/indexspeed_test.go
@@ -21,7 +21,6 @@ import (
 	librarian "github.com/ssbc/margaret/indexes"
 	"github.com/ssbc/margaret/multilog"
 
-	"github.com/ssbc/go-ssb/internal/testutils"
 	"github.com/ssbc/go-ssb/repo"
 	refs "github.com/ssbc/go-ssb-refs"
 	"github.com/ssbc/go-ssb-refs/tfk"
@@ -77,10 +76,6 @@ func BenchmarkIndexFixturesUserFeeds(b *testing.B) {
 func TestIndexFixtures(t *testing.T) {
 	r := require.New(t)
 	a := assert.New(t)
-
-	if testutils.SkipOnCI(t) {
-		return
-	}
 
 	f, err := os.Open("v2-sloop-authors.json")
 	r.NoError(err)

--- a/multilogs/integration_prep.bash
+++ b/multilogs/integration_prep.bash
@@ -16,8 +16,10 @@ test "$dest" != "" || {
 sha256sum -c v2-sloop-m100000-a2000.tar.gz.shasum || {
     wget "https://github.com/ssb-ngi-pointer/ssb-fixtures/releases/download/2.3.0/v2-sloop-m100000-a2000.tar.gz"
 
-    echo 'rerun me'
-    exit 0
+    sha256sum -c v2-sloop-m100000-a2000.tar.gz.shasum || {
+        echo 'download of ssb-fixtures failed'
+        exit 1
+    }
 }
 
 

--- a/multilogs/integration_prep.bash
+++ b/multilogs/integration_prep.bash
@@ -27,4 +27,4 @@ rm -r testrun
 mkdir -p tmp/unpack
 tar xf v2-sloop-m100000-a2000.tar.gz -C tmp/unpack
 
-go run go.cryptoscope.co/ssb/cmd/ssb-offset-converter -if lfo tmp/unpack/flume/log.offset $dest
+go run ../cmd/ssb-offset-converter -if lfo tmp/unpack/flume/log.offset $dest

